src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,88 @@
+package duke.task;
+
+/**
+ * The abstract base class representing a task.
+ *
+ * @author Win Sheng
+ * @since 3 September 2023
+ */
+public abstract class Task {
+    public String description;
+    public boolean isDone;
```

Comment:
I think all your variables involving booleans are named correctly with the is prefix, well done! 👍